metals/src/main/scala/scala/meta/internal/builds/MillBuildTool.scala

```diff
@@ -35,6 +35,7 @@ case class MillBuildTool(userConfig: () => UserConfiguration)
       version
     }
     val cmd = List(
+      "-i", // In some environments (such as WSL or cygwin), mill must be run using interactive mode (-i)
```

Comment:
Could only add it if ` (Properties.isWin)` ? Something like:
```
val iOption =  if (Properties.isWin) List("-i") else List()}
val cmd = iOption :: "--predef" ::  predefScriptPath(millVersion).toString ::  "mill.contrib.Bloop/install"
```
?

Reply:
Good point. I'll make these changes